frontend/src/index/index.scss

```diff
@@ -18,12 +18,12 @@ code {
   margin-bottom: 1px;
 }
 
-.landing-page {
+.page {
     padding: 55px 55px 120px;
```

Comment:
SASS allows us to define variables, and when we're doing things like clearing a header or footer it's very useful to, e.g., set a `$header-height` var instead of using magic numbers. That'll make this stylesheet more readable/usable for future maintainers.